src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+// Перевіряємо, чи існує таблиця
+if (table) {
+  const rows = Array.from(table.rows);
+
+  // Створюємо DocumentFragment для уникнення переробок DOM
+  const fragment = document.createDocumentFragment();
+
+  rows.forEach((row) => {
+    const secondCell = row.cells[1]; // Отримуємо другу клітинку
+
+    if (secondCell) {
+      const clonedCell = secondCell.cloneNode(true); // Клонуємо другу клітинку
+
+      // Вставляємо клон перед останньою клітинкою
+      row.insertBefore(clonedCell, row.cells[row.cells.length - 1]);
+    }
+  });
+
+  // Додаємо змінений фрагмент до таблиці
+  table.appendChild(fragment);
```

Comment:
As mentioned before, you should append the cloned cells to the fragment, and then append the fragment to the table. Here, you are appending an empty fragment to the table, which has no effect.